### PR TITLE
reddit: adding praw compatibility

### PR DESCRIPTION
**Pull request by RustyBower**
praw 4.0.0 changes get_redditor() to redditor(). This patch fixes sopel's reddit module for all versions of praw until a permanent fix can be implemented in sopel 7.

### CODE REVIEW

File: sopel/modules/reddit.py [hunk 1]
```diff
@@ -124,6 +124,7 @@ def redditor_info(bot, trigger, match=None):
     try:
         u = r.get_redditor(match.group(2))
     except Exception:  # TODO: Be specific
+        u = r.redditor(match.group(2))
```
Comment (dgw) [inline sopel/modules/reddit.py]: This won't do anything… This `except` block will always `return` no matter what, because of the `if`/`else` below it.

**Comment by dgw** [comment]
> Will […] leave correcting the patch itself to @RustyBower.

I lied there. @RustyBower let me know if this is what you meant to do (and if not, amend again).

**Comment by RustyBower** [comment]
That looks good!

**Comment by Exirel** [comment]
77 commits? I think there is something deeply wrong with this one.

**Comment by dgw** [comment]
@RustyBower This was OK… what did you force push for?

**Comment by RustyBower** [comment]
Not sure. I think it got caught up with the 7 branch when I pushed to that? Doesn't look like the code changed.

**Comment by dgw** [comment]
I'll re-push my branch when I get home, I guess. Then @Exirel can review?

**Comment by dgw** [comment]
Well, I forgot to re-push this yesterday, but better late than never? :S

**Comment by Exirel** [comment]
👍

**Comment by dgw** [comment]
Quick, merge before Rusty overwrites the branch again! :stuck_out_tongue:

**Pull request merged**